Added Task 3

PNVB opened the pull request.

PiTBNV committed:
Finished task 3, created DataBase, writed queries for selecting data

PiTBNV committed:
changed coding

changed coding from Windows-1251 to UTF-8 so cyrillic shows correctly

PNVB committed:
Create diagram

kuzkok commented:
По диаграмме:
Не используй "-" в наименованиях полей\значениях перечислений - оно не обрабатывается, можно в Caption положить
УчастокСети сделай Композицией

PiTBNV committed:
modified one link to composition, edited enumeration list

PiTBNV committed:
Added Task5

Created SQL-querry for creating database according to UML-design. Generated .NET App using FlexberryDesign.

PNVB committed:
Update MSSql.create.sql

Changed encoding

PNVB committed:
Update Task5: .Net App

PNVB committed:
Create Task6

Created project for Task6, added business-server to diagram, wrote 2 algorithms

PNVB committed:
Added NotStored field in Network class

PNVB committed:
some fixes

Changed the link between Building and ThermoObject to composition, added the function that calculate the sum of all ThermoObject's squares in building

PNVB committed:
Create ОбъектыТеплопотребления.crp

kuzkok commented:
Нужны доработки, подробности в письме, этот помержу.
